polynote-frontend/polynote/ui/display/display_content.ts

```diff
@@ -6,16 +6,109 @@ import {Content, details, div, span, tag, TagElement} from "../tags";
 import {ArrayType, MapType, OptionalType, StructField, StructType} from "../../data/data_type";
 import embed from "vega-embed";
 
+const frames: Record<number, HTMLIFrameElement> = {};
```

Comment:
Maybe it would be better if this was scoped to each notebook?

Reply:
I'm not sure what the practical benefit would be, other than guilt about global state in JavaScript-land. This is a big stupid workaround to begin with; I figured there was no point in pretending it wasn't 😛